src/ui/ship_building/ship_building_view.gd

```diff
@@ -26,9 +26,9 @@ func _ready():
 	ScreenTransitionManager.fade_in(1.5)
 	await ScreenTransitionManager.transitioned
 
-	bgm_music = load("res://assets/audio/music/ld54-bgm-medley-no-alarms-1.1.mp3")
-	bgm_audio_player = SoundManager.play_music(bgm_music, 0.2, "Music")
-	bgm_audio_player.finished.connect(play_bgm_again)
+	# bgm_music = load("res://assets/audio/music/ld54-bgm-medley-no-alarms-1.1.mp3")
+	# bgm_audio_player = SoundManager.play_music(bgm_music, 0.2, "Music")
+	# bgm_audio_player.finished.connect(play_bgm_again)
```

Comment:
If  FMOD is taking over control of these audio cues then just remove these lines if they aren't needed.